### PR TITLE
Portfolio second

### PR DESCRIPTION
**Pull request by binodbhusal**
-This is the second branch of my portfolio second. In this branch, I have made changes following.
1. Added cards by implementing grid and flexbox. 
2. Created hover buttons.
3. updated about my self section.
4. inserted different images

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,199 @@
...
+    <ul id="second_list_D">
+        <li id="second_list1D">HTML</li>
+        <li id="second_list2D">CSS</li>
+        <li id="second_list3D">JAVA SCRIPT</li>
+    </ul>
+
+    <button class="btn_project">See Project</button>
+</div>
+</div>
+</div>
```
Comment (mahbubul14) [inline index.html]: - [ ] The container of the section is a `section` tag. For the works section, all elements of the work section should be wrapped with a `section` tag and kindly match your design to the Figma design. :+1: 

Your design             |  Figma Design
:-------------------------:|:-------------------------:
yourdesign_work | figma_work

File: index.html [hunk 2]
```diff
@@ -0,0 +1,199 @@
...
+    <div class="frame">
+        <p>Frameworks </p><i class="fa-solid fa-chevron-right"></i>   
+    </div>
+    
+    <div class="skills">
+       <p>Skills </p><i class="fa-solid fa-chevron-right"></i>
+  
+    </div>
+    
+</div>
```
Comment (mahbubul14) [inline index.html]: - [ ] The container of the section is a `section` tag. Kindly try to match the Figma design as well. 

Your design             |  Figma Design
:-------------------------:|:-------------------------:
yourdesign_myself | figma_myself

**Review by mahbubul14 — CHANGES_REQUESTED** [review]
Hi @binodbhusal,

Good job so far! :+1: 
There are some issues that you still need to work on to go to the next project but you are almost there!

# Required Changes ♻️

- [ ] Kindly check the inline comments under the review. :arrow_down: 

###  To Highlights
:heavy_check_mark: No linter error.
:heavy_check_mark: Proper Github flow used.
:heavy_check_mark: Descriptive PR title & description.

### Optional suggestions

Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better.

- [ ] The commit message should be written in the imperative tense and the first letter should also be capitalized. `Ex: "Fix bug" instead of "fixed bug".` [For more details...](https://www.freecodecamp.org/news/level-you-up-to-awesome-commit-messages-a85558cb90e8)


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification. You can also reach me on [slack](https://microverse-students.slack.com/team/U01J39GL614)

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

**Review by Owusu-Desmond — APPROVED** [review]
Hi @binodbhusal , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
📌 No Linters error ✅
📌 Descriptive pull request✅
📌 Professional README file ✅

## Optional suggestions

- [ ] As requested by the previous code reviewer :warning: The commit message should be written in the imperative tense and the first letter should also be capitalized. Ex: "Fix bug" instead of "fixed bug". [For more details...](https://www.freecodecamp.org/news/level-you-up-to-awesome-commit-messages-a85558cb90e8)

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**